draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
Even if you used hash commitment for this, then a persistent MitM will be able to learn the recovery token and then use it.  Any recovery needs to use asymmetric crypto that is bound to a specific request, or be out of band entirely.